flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/DeclarativeSlotManager.java

```diff
@@ -79,8 +81,19 @@ public class DeclarativeSlotManager implements SlotManager {
     private final Map<JobID, String> jobMasterTargetAddresses = new HashMap<>();
     private final Map<SlotID, AllocationID> pendingSlotAllocations;
 
+    /** Delay of the requirement change check in the slot manager. */
+    private final Duration requirementsCheckDelay;
```

Comment:
I tend to move this to a preceding issue.

Reply:
do you mean create another ticker for this ?

Reply:
Yes.

Reply:
Get it, will do it

Reply:
A preceding PR https://github.com/apache/flink/pull/19886 is opened. Please take a look, thanks